vars/withSelenium.groovy

```diff
@@ -67,7 +67,7 @@ void call(Map config = [:], String seleniumNetwork, Closure closure) {
         def chromeContainers = runWorkerNodes("${config.workerImageChrome}:${config.seleniumVersion}", networkParameter, gridDebugParameter, seleniumIp, config.chromeWorkerCount)
 
         try {
-            def seleniumHubHost = "${seleniumIp}:${config.hubPortMapping}"
+            def seleniumHubHost = "localhost:${config.hubPortMapping}"
```

Comment:
This localhost replcament looks like a critical change. Does this work generally and specifically?